e2e/test/scenarios/admin/datamodel/metrics.cy.spec.js

```diff
@@ -324,5 +324,6 @@ describe("scenarios > admin > datamodel > metrics", () => {
 });
 
 function selectTable(tableName) {
+  cy.findByText("Select a table").click();
```

Comment:
ESLint isn't complaining only because this line is in a separate function. If we inline this function, it will.

```suggestion
  cy.get(".GuiBuilder").findByText("Select a table").click();
```